Sources/XCTQueues/TestQueueDriver.swift

```diff
@@ -0,0 +1,61 @@
+import Queues
+import Vapor
+
+extension Application.Queues.Provider {
+    public static var test: Self {
+        .init {
+            $0.queues.use(custom: TestQueuesDriver())
+        }
+    }
+}
+
+public struct TestQueuesDriver: QueuesDriver {
+    public func makeQueue(with context: QueueContext) -> Queue {
+        TestQueue(context: context)
+    }
+    
+    public func shutdown() {
+        // nothing
+    }
+}
+
+public struct TestQueue: Queue {
+    public static var queue: [JobIdentifier] = []
```

Comment:
Could we use `app.storage` here instead of static variables? That would prevent the need for doing `TestQueue.reset` as well as allow for tests on separate applications to run concurrently without conflicting.